webpack.prod.js

```diff
@@ -0,0 +1,6 @@
+const { merge } = require('webpack-merge');
```

Comment:
As recommended in [the documentation](https://webpack.js.org/guides/production/).